sylt/src/formatter.rs

```diff
@@ -54,16 +54,34 @@ fn write_parameters<W: Write>(
     Ok(())
 }
 
-fn write_blob_fields<W: Write>(
+fn write_blob_fields<T: Clone, W: Write>(
```

Comment:
I can't add a suggestion here since there's mixed additions and deletions, so consider the following diff:
```diff
diff --git a/sylt/src/formatter.rs b/sylt/src/formatter.rs
index 235cd22..afa390f 100644
--- a/sylt/src/formatter.rs
+++ b/sylt/src/formatter.rs
@@ -54,10 +54,10 @@ fn write_parameters<W: Write>(
     Ok(())
 }

-fn write_blob_fields<T: Clone, W: Write>(
+fn write_blob_fields<T, W: Write>(
     dest: &mut W,
     indent: u32,
-    fields: Vec<(String, T)>,
+    mut fields: Vec<(String, T)>,
     sub_write: fn(&mut W, u32, T) -> fmt::Result,
 ) -> fmt::Result {
     write!(dest, " {{ ")?;
@@ -66,7 +66,7 @@ fn write_blob_fields<T: Clone, W: Write>(
             write!(dest, " }}")?;
         }
         1 => {
-            let (field, expr) = fields[0].clone();
+            let (field, expr) = fields.remove(0);
             write!(dest, "{}: ", field)?;
             sub_write(dest, indent, expr)?;
             write!(dest, " }}")?;
```
Since we're using owned data we shouldn't need to clone anywhere.